lib/board.rb

```diff
@@ -1,5 +1,7 @@
 # Board Class
 class Board
+  attr_accessor :board
+
```

Comment:
Think extensively about this. This allows any foreign object to be able to write the array where you are storing the board state. E.g.
```
class MyClass
  b = Board.new
  b.board = 'Will this break if I assign a string?'
```